### ISSUE TITLE
CREO2URDF – Add in idyntree the possibility to export xml blobs and sensors to the urdf

### ISSUE THREAD

**Issue by Nicogene** [icub-tech-iit/creo2urdf#24]
### Task description

This task requires to make changes to idyntree:
```[tasklist]
### Tasks
- [ ] https://github.com/robotology/idyntree/issues/1080
```

### Definition of done

Xml blobs and sensors have been exported

cc @traversaro @pattacini

**Comment by Nicogene** [comment]
x-ref:
- https://github.com/robotology/idyntree/pull/1088

**Comment by Nicogene** [comment]
Thanks to @traversaro is now possible to retrieve directly from the `iDynTree::Model` the list of sensors, see:

- https://github.com/robotology/idyntree/pull/1106

**Comment by Nicogene** [comment]
Today I started working on moving the logic for exporting the urdf of the sensors inside idyntree instead on doing it on the creo2urdf-side, but I discovered that I am missing several information that requires further changes/refactoring on idyntree side.

In particular I miss;
- `frame`
- `measure_direction`
- `sensorBlobs`

The first two can be "deducted" from other informations already carried by the `iDynTree::SixAxisForceTorqueSensor` but the latter requires changes in the class itself, in the parsing stage (How we distinguish a xmlblob to something else inside the xml?) and finally in the export stage.

Since on creo2urdf side this operation is quite simple because we directly got these informations from the yaml, with @mfussi66 agreed that work for introducing this feature in idyntree is not worthy and maybe it goes also out of scope of what a sensor is inside idyntree, because usually the xml blobs are gazebo-related.

Given that we can define this activity concluded already in 
- https://github.com/robotology/idyntree/pull/1088 
- #46 

cc @pattacini @traversaro
